src/gtfs/read.rs

```diff
@@ -396,15 +388,23 @@ where
             )
         }
     }
-    collections.stop_time_headsigns = headsigns;
 
     for (vj_idx, mut stop_times) in tmp_vjs {
         stop_times.sort_unstable_by_key(|st| st.stop_sequence);
+        stop_times.dedup_by(|st2, st1| {
+            let is_same_seq = st2.stop_sequence == st1.stop_sequence;
+            if is_same_seq {
+                warn!(
+                    "remove duplicated stop_sequence '{}' of vehicle '{}'",
```

Comment:
It is about GTFS reading so Vehicle Journey doesn't exist.

```suggestion
                    "remove duplicated stop_sequence '{}' of trip '{}'",
```